AK/Bitmap.h

```diff
@@ -171,14 +171,14 @@ class Bitmap {
         if (!first_index.has_value())
             return {};
 
-        size_t free_region_start = first_index;
-        size_t free_region_size = 1;
+        size_t free_region_start = 0;
+        size_t free_region_size = 0;
```

Comment:
Hmm, why is this logic being changed? Should it not be like this:
```suggestion
        size_t free_region_start = first_index.value();
        size_t free_region_size = 1;
```

Reply:
I did change the logic. I thought it would be a little bit cleaner, because the loop will set the right values in the first if.

If it’s better to return everything on its place, I will)

Reply:
But the loop doesn't start at `0`, it starts at `first_index + 1`, so it may never find the same values.

Also you are changing the loop start from `first_index + 1` to `first_index`. The point of this code is that we've already found the first index in the call to `find_first_unset()` above and we want to continue accumulating *after* the first unset bit.